src/cts/test/lvt_lib.tcl

```diff
@@ -0,0 +1,23 @@
+source "helpers.tcl"
+source "cts-helpers.tcl"
+
+read_liberty Nangate45/Nangate45_typ.lib
+read_liberty Nangate45/Nangate45_lvt.lib
+read_lef Nangate45/Nangate45.lef
+read_lef Nangate45/Nangate45_lvt.lef
+
+set block [make_array 300 200000 200000 150]
```

Comment:
block does not seem used.

Reply:
We need a block definition.  Absolute coordinates don't matter but a block needs to be defined.

Reply:
Is the tcl variable used inside subsequent commands? Or just make_array create the block in the database?
Normally the block is created when reading netlist, right?

Reply:
> Is the tcl variable used inside subsequent commands? Or just make_array create the block in the database? Normally the block is created when reading netlist, right?

Correct.  When you do read_def or read_verilog, block gets created as part of the netlist reading.  When a design is created using ODB APIs like this testcase, you need to create a block manually.

Reply:
So block is not needed? The line "make_array 300 200000 200000 150" is enough, right?
Btw why make_array and not make_block?